pkgs/servers/rustdesk-server/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, libsodium
+, sqlite
+, nix-update-script
+, testers
+, rustdesk-server
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rustdesk-server";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "rustdesk";
+    repo = "rustdesk-server";
+    rev = version;
+    hash = "sha256-KzIylbPe+YN513UNFuisvDAIe4kICGYDUrfURDOw/no=";
+  };
+
+  cargoLock = {
```

Comment:
Why not specify the cargo hash rather than vendoring the file and adding the hashes for the git deps?

Reply:
It seems I cannot specify outputHashes without specifying a local lock file.

Reply:
@dasJ I can confirm @gaelreyrol statement here, tried to build it by specifying the `cargoSha256` and the build fails with the following:

```
last 10 log lines:
       > building
       > source = "git+https://github.com/open-trade/async-speed-limit#f89f702ae01d4016429543d2f0dda1086157e420"
       > source = "git+https://github.com/open-trade/confy#630cc28a396cb7d01eefdd9f3824486fe4d8554b"
       > ERROR: The Cargo.lock contains git dependencies
       >
       > This is currently not supported in the fixed-output derivation fetcher.
       > Use cargoLock.lockFile / importCargoLock instead.
```

Looks like the **rustdesk** application is also using hashes for the git deps, see most recent [commit](https://github.com/NixOS/nixpkgs/commit/cf3c4da2ea84369ba2bdc8c412de01ddcb91d75c).

Reply:
Damn.